### PR TITLE
Added microservice descriptions and commands to documentation 

### PR DESCRIPTION
**Pull request by erg9836**
updated documentation by adding descriptions and command examples to new microservice entries.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -444,6 +472,12 @@ To view the specific ffmpeg encoding options for each file, view the sourcecode
     * mmtest does not test dvd derivatives, as iso file checksums cannot match consistently.
 * To run mmtest, type: `mmtest`
 
+#### paperingest
+* This is an interactive script that asks you to enter your name and the media id. After you enter the info, the scanner will begin to scan your papers. To use this script, type the following: 'paperingest'.
+
+#### phasecheck
+* this script performs a series of quality assurance tests on an audio visual file or set of files.
```
Comment (CSchloss385) [inline README.md:479]: Just add that this microservice makes sure the audio is in phase.

**Review by CSchloss385 — COMMENTED** [review]
Mainly just needs some fixes to capitalization and some additional description.

File: README.md [hunk 2]
```diff
@@ -253,22 +253,29 @@ Across all mediamicroservices, you can always receive the usage information by t
...
-* Finally, use -v as an option if you want to fully verify checksums. If no checksums exist, the script will create the initial ones. Verification will version existing checksums by adding the date they were created to the filename and create new ones, and log the difference to a checksumprocess log, which will be placed in the metadata directory of the package. To use -v, type  `checksumpackage -v [input]`
+* Finally, use -v as an option if you want to fully verify checksums. If no checksums exist, the script will create the initial ones. Verification will version existing checksums by adding the date they were created to the filename and create new ones, and log the difference to a checksumprocess log, which will be placed in the metadata directory of the package. To use -v, type  `checksumpackage -v [input]`.
+
+#### createmmdb
+* createmmdb configures a database, users and login profiles. Your command will look like this 'Usage: -c (create database) -u (create user) -h (help)'.
```
Comment (kieranjol) [inline README.md:278]: Should the usage use backticks instead of quotes?

File: README.md [hunk 3]
```diff
@@ -283,16 +290,19 @@ To view the specific ffmpeg encoding options for each file, view the sourcecode
...
+* fix\_left2stereo takes an input video file or files and produces outputs that map the left channel of the input to a stereo mix in the output. The suffix "\_left2stereo" is added to the filename to distinguish it from the original. Your command will look like this: `fix_left2stereo [file1] [file2]`.
 
 #### fix_rewrap
-* fix\_rewrap takes an input video file or files and produces outputs through re-multiplexing all tracks of the input into a new container. The suffix "\_rewrap" is added to the file name to distinguish it from the original. Your command will look like this: `fix_rewrap [file1] [file2]`
+* fix\_rewrap takes an input video file or files and produces outputs through re-multiplexing all tracks of the input into a new container. The suffix "\_rewrap" is added to the file name to distinguish it from the original. Your command will look like this: `fix_rewrap [file1] [file2]`.
+
+#### fix_timecode
+* fix\_timecode takes an input video file(s) and produces output that sets the timecode to 00:00:00;00. Your command will look like this: 'fix_timecode [file1]'.
```
Comment (kieranjol) [inline README.md:299]: backticks again?

File: README.md [hunk 4]
```diff
@@ -310,6 +320,9 @@ To view the specific ffmpeg encoding options for each file, view the sourcecode
 <img src="https://github.com/mediamicroservices/mm/blob/master/Resources/ingestfile.gif" width="500">
 <img src="https://github.com/mediamicroservices/mm/blob/master/Resources/ingestfilegui.gif" width="500">
 
+#### makeaudiographs
+* This script creates a visual representation of an audio file. Another helpful tool for audio QC. To use this script, run the following command: 'makeaudiographs [file]'.
```
Comment (kieranjol) [inline README.md:324]: backticks (I'll stop now :)

**Review by kieranjol — COMMENTED** [review]
Just a few comments on quotes used instead of backticks for code sections, I didn't flag all examples.

**Review by kieranjol — COMMENTED** [review]
Backticks look good to me!

**Comment by erg9836** [comment]
> Backticks look good to me!

Thanks!

**Pull request merged**